api_docs/drone_doc.py

```diff
@@ -0,0 +1,212 @@
+"""Generated API Documentation for Drone API using drone_doc_gen.py."""
+
+drone_doc = {
+    "@context": "http://hydrus.com/contexts/droneapi.jsonld",
+    "@id": "http://hydrus.com/droneapi/vocab",
+    "@type": "ApiDocumentation",
+    "description": "API Documentation for the drone side system",
+    "entrypoint": "/droneapi/",
+    "possibleStatus": [],
+    "supportedClass": [
+        {
+            "@context": "http://hydrus.com/context/Drone.jsonld",
+            "@id": "http://hydrus.com/droneapi/vocab/Drone",
+            "@type": "Class",
+            "description": "Class for a drone",
+            "supportedOperation": [
+                {
+                    "@id": ":change_speed",
+                    "@type": "Operation",
+                    "expects": "http://auto.schema.org/speed",
+                    "method": "PUT",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Speed Changed"
+                        }
+                    ],
+                    "returns": "null",
+                    "title": "ChangeSpeed"
+                },
+                {
+                    "@id": ":change_position",
+                    "@type": "Operation",
+                    "expects": "http://schema.org/geo",
+                    "method": "PUT",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Position Changed"
+                        }
+                    ],
+                    "returns": "null",
+                    "title": "ChangePosition"
+                },
+                {
+                    "@id": ":change_status",
+                    "@type": "Operation",
+                    "expects": "http://schema.org/status",
+                    "method": "PUT",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Status Changed"
+                        }
+                    ],
+                    "returns": "null",
+                    "title": "ChangeStatus"
+                },
```

Comment:
I'll modify everything into a status object, and we can send a new status object every time we need to modify some feild or the other. Something similar to receive_order I think